### ISSUE TITLE
Remove FontAwesome?

### ISSUE THREAD

**Issue by jpdevries** [modmore/QuickstartButtons#26]
Now that modmore doesn't support MODX 2.2 and 2.3 ships with FontAwesome, should we remove FontAwesome?

**Comment by Mark-H** [comment]
I don't think we should be actively breaking compatibility, but when we reach a next minor and 2.2 support is "holding us back" I think it makes sense to remove it if it means we can simplify some code.

**Comment by jpdevries** [comment]
curious if maybe we should only load it for pre 2.3 then (like we do with other assets in Redactor). Currently it "double loads" in 2.3

**Comment by Mark-H** [comment]
Given the icons selection is based on the css file, I'm not sure we should get rid of it just yet. We're also using a different prefix (the standard `fa` instead of `icon` in MODX), so for now let's just leave it in.

Might be a good one for QSB 2. Or MODX 3. Which ever comes first.
